examples/js/loaders/MMDLoader.js

```diff
@@ -67,6 +68,23 @@ THREE.MMDLoader.prototype.defaultToonTextures = [
 	'data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACAAAAAgCAYAAABzenr0AAAAL0lEQVRYR+3QQREAAAzCsOFfNJPBJ1XQS9r2hsUAAQIECBAgQIAAAQIECBAgsBZ4MUx/ofm2I/kAAAAASUVORK5CYII='
 ];
 
+/*
+ * getImageData() of CanvasRenderingContext2D with image in other domain
+ * for transparency detection fails because of cross-origin problem
+ * even if server responds with "Access-Control-Allow-Origin: *".
+ * If this.imageCrossOrigin is true, MMDLoader tries to solve this issue
+ * with
+ *   var image = new Image();
+ *   image.crossOrigin = 'anonymous';
+ *   image.src = url;
+ * This workaround can be removed if ImageLoader supports crossOrigin.
```

Comment:
This new API seems avoiding the cross origin issue but I've faced

three.js:19115 DOMException: Failed to execute 'texImage2D' on 'WebGLRenderingContext': The cross-origin image at blob:null/8074b93f-ff43-46be-9339-e81f20a88114 may not be loaded.
texImage2D @ three.js:19115
uploadTexture @ three.js:17989
setTexture2D @ three.js:17667
setTexture2D @ three.js:22009
setValueT1 @ three.js:4596
WebGLUniforms.upload @ three.js:4923
setProgram @ three.js:21448
WebGLRenderer.renderBufferDirect @ three.js:20224
renderObjects @ three.js:21015
WebGLRenderer.render @ three.js:20752
render @ OutlineEffect.js:441
render @ MMDLoader.js:2748
render @ webgl_loader_mmd3.html:247
animate @ webgl_loader_mmd3.html:235

I'll see the detail later...